[FIX] doc: unclear report actions fields and report tag attributes. (#57923)
odoo/odoo: doc/reference/actions.rst

``report_type`` (default=qweb-pdf)
    either ``qweb-pdf`` for PDF reports or ``qweb-html`` for HTML
``report_name`` (mandatory)

>     the name (:term:`external id`) of the qweb template used to render the report

Reviewer: Cool, I did not know about the glossary.

``name`` (mandatory)

>     If ``print_report_name`` isn't specified, will be used as file name.

Reviewer 2: ```suggestion
    used as the file name if ``print_report_name`` is not specified.
```
